src/uint256.h

```diff
@@ -134,6 +139,11 @@ class uint256 : public base_blob<256> {
 public:
     uint256() {}
     explicit uint256(const std::vector<unsigned char>& vch) : base_blob<256>(vch) {}
+
+    static uint256 TryFromArray(const std::array<unsigned char, 32>& vch) {
```

Comment:
`FromArray` since it's not fallible, and you can also just template this under base_blob, I'd imagine.